lib/routes/account.js

```diff
@@ -567,6 +569,20 @@ module.exports = function (
                       events: events.length
                     })
                   }
+
+                  // IP Profiling sets a flag to bypass sign-in confirmation if this
+                  // user has verified a session with-in the past day from this ip address.
+                  if (ipProfilingEnabled && verified && recency === 'day') {
+                    hasVerifiedSessionFromIp = true
+
+                    log.info({
+                      op: 'Account.ipprofiling.seenAddress',
+                      uid: emailRecord.uid.toString('hex')
+                    })
+
+                  } else {
+                    hasVerifiedSessionFromIp = false
```

Comment:
also maybe set `hasVerifiedSessionFromIp = false` above and avoid the `else`? but we add logging the `else` could be fine?